SpywareFilter/sections/specific.txt

```diff
@@ -22,6 +22,9 @@
 ||nd.demdex.net/event?d_mid=$redirect=nooptext,important,domain=9now.com.au
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 !
+||dtory.ebs.co.kr/wlo^
```

Comment:
I'm not sure about this one. I don't see it when the script `home.ebs.co.kr/common/js/wlo.js` is blocked.

Reply:
That is for to block the server's logging server if they change their code.
The sending command is included in `https://home.ebs.co.kr/common/js/wlo.js`.

Reply:
Hm, in this case, ok.